andela/pippin-ah-backend
# #162727544 Configure houndci

#### What does this PR do?
This PR integrates houndci with the repo

#### Description of Task to be completed?
- Add .houndci.yml file to the root directory
- Add configuration code to .houndci.yml file
- Add houndci badge to readme.md
- Delete package-lock.json
- Add package-lock.json to .gitigonre

#### How should this be manually tested?
- View readme.md to see houndci badge
- Push a new branch to GitHub and expect to not have package-lock.json in the remote branch
- Open a GitHub pull request
- Hound will review the pull request and comment inline on any style violation as declared in .eslint file

#### What are the relevant pivotal tracker stories?
The relevant pivotal tracker story for this PR:
 - #162727544 https://www.pivotaltracker.com/n/projects/2229853/stories/162727544
